### PR TITLE
gssh: depends_on openjdk@11

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

relates to https://github.com/int128/groovy-ssh/commit/b76ca69c7b2a2cf0bacaed59afa35208cbbfbb1a

### CODE REVIEW

File: Formula/gssh.rb [hunk 1]
```diff
@@ -13,13 +14,13 @@ class Gssh < Formula
...
 
-  depends_on java: "1.8"
+  depends_on "openjdk@11"
 
   def install
     ENV["CIRCLE_TAG"] = version
     system "./gradlew", "shadowJar"
     libexec.install "cli/build/libs/gssh.jar"
-    bin.write_jar_script libexec/"gssh.jar", "gssh", java_version: "1.8"
+    bin.write_jar_script libexec/"gssh.jar", "gssh", java_version: "11"
```
Comment (SMillerDev) [inline Formula/gssh.rb:23]: java_version is not needed

Comment (chenrui333) [reply Formula/gssh.rb:23]: hmm, we dont have to specify `java_version` about which version to pick?

Comment (SMillerDev) [reply Formula/gssh.rb]: only if it's a cask AFAIK

Comment (chenrui333) [reply Formula/gssh.rb]: that is right, yeah, this is compiled code, which already targets specific java binary format. Make sense.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**